providers/nmea0183-signalk.js

```diff
@@ -14,81 +14,56 @@
  * limitations under the License.
  */
 
-/* Usage: this is the pipeElement that transforms NMEA0183 input to Signal K deltas
```

Comment:
Why delete the documentation?

Reply:
I set out to rewrite it, and then forgot about it. I'll add it back in.